### PR TITLE
[10.0] Create module crm_lead_partner_role

### PR DESCRIPTION
**Pull request by grindtildeath**

### CODE REVIEW

**Comment by grindtildeath** [comment]
@sbidoul I cannot find any up to date explanation on what's to do now with the new README.rst generation feature (https://github.com/OCA/maintainer-tools/pull/339).
It seems https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#readme is still to be updated.
However, I did add `readme` folder with `DESCRIPTION.rst` and `CONTRIBUTORS.rst` as explained on  https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst but travis now complains there's no README.rst. Should I simply add an empty file ? Is there anything I am missing somewhere ?

**Comment by sbidoul** [comment]
@grindtildeath yes a dummy README.rst is fine, it will be overwritten nightly

File: crm_lead_partner_role/models/crm_partner_role.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import models, fields
+
+
+class CrmPartnerRole(models.Model):
+    _name = 'crm.partner.role'
```
Comment (simahawk) [inline crm_lead_partner_role/models/crm_partner_role.py:8]: _description

File: crm_lead_partner_role/models/crm_role.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import models, fields
+
+
+class CrmRole(models.Model):
+
```
Comment (simahawk) [inline crm_lead_partner_role/models/crm_role.py:8]: _description

File: crm_lead_partner_role/models/crm_lead.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+# Copyright 2018 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import models, fields
+
+
+class CrmLead(models.Model):
+
+    _inherit = 'crm.lead'
+
+    partner_role_ids = fields.One2many(
```
Comment (simahawk) [inline crm_lead_partner_role/models/crm_lead.py:11]: help="Partners involved on this lead and their roles." (or alike)

File: crm_lead_partner_role/readme/DESCRIPTION.rst [hunk 4]
```diff
@@ -0,0 +1,6 @@
+This modules extends the features of crm modules and allows to add partner
+roles.
+A new 'Related people' tab will appear on leads form view, allowing to add
+multiple partners with a defined role for each one.
+This module can then be extend to process these data when converting to
```
Comment (simahawk) [inline crm_lead_partner_role/readme/DESCRIPTION.rst]: extended

**Review by simahawk — CHANGES_REQUESTED** [review]

**Review by simahawk — APPROVED** [review]

**Comment by fclementic2c** [comment]
Functional test Ok
:+1:

**Comment by simahawk** [comment]
@sbidoul @grindtildeath I still see 
```
2018-06-12 08:01:21,841 193 WARNING openerp_test odoo.addons.base.module.module: module crm_lead_partner_role: description is empty !
```
Linting should be fixed too ;)

**Review by tschanzt — APPROVED** [review]

**Review by rafaelbn — COMMENTED** [review]
Hi @grindtildeath @simahawk @tschanzt ,

I have been testgin in runbot :red_circle: (we should ensure runbot is green :green_heart: ) and I was trying to find where a can Assign partner and roles to your leads... Could you please tell me how?

Thanks

**Comment by grindtildeath** [comment]
@rafaelbn as written in the readme there's a new tab on crm.lead:
image

**Comment by rafaelbn** [comment]
OK thanks. Then please review Readme as it looks it doesn't generate well and after is inaccesible as you can see in my screenshot. I've tested the module 👍 
So just the Readme for merging! :-)

**Comment by rafaelbn** [comment]
Ups, also please review bots! Are 🔴

**Review by rafaelbn — APPROVED** [review]

**Pull request closed without merging**